src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,27 @@
  */
 trait VarDumperTestTrait
 {
+    /** @internal  */
```

Comment:
extra space :)
should be on several lines, or `@after` should be on one?

Reply:
Oops. Fixed